src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File readFile = new File(fromFileName);
+        File writeFile = new File(toFileName);
+
+        String[] infoArr = readFromFile(readFile).toString().split("[\\r\\n\\s]+");
+
+        int buy = 0;
+        int supply = 0;
+        String buyPrefix = "buy,";
+        String supplyPrefix = "supply,";
+        int buyIndex;
+        int supplyIndex;
+
+        for (String info : infoArr) {
+
+            if (info.startsWith(buyPrefix)) {
+                buyIndex = Integer.parseInt(info.substring(buyPrefix.length()));
```

Comment:
[Use split() instead of substring()](https://github.com/mate-academy/jv-work-with-file/blob/master/checklist.md#use-split-instead-of-substring)